src/ontology/edits/oeo-model.omn

```diff
@@ -1190,7 +1190,7 @@ Class: Support
         SoftwareMaintenance
     
     
-Class: SynthethicDataset
+Class: SyntheticDataset
 
     Annotations: 
         <http://purl.obolibrary.org/obo/IAO_0000112> "The projected power usage during an idealized model year."@en,
```

Comment:
In the Annotation below (line 1197) there is another typo: 'obtain' but 'optain'. I cannot fix it here directly...